index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My porfolio</title>
+</head>
+<body>
+    <header>
+        <menu class="nav-container">
+            <h1><a href="#">Nana Antwi</a></h1>
+            <nav>
+                <img class="nav-icon" src="images/Union.png" alt="">
+            </nav>
+        </menu>
+    </header>
+    <section class="introduction">
+        <h1>
+            Hey There. I’m Nana
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <div class="social-icons">
+            <ul>
+                <li><a href="#"><img src="images/Vector4.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Group.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector2.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector1.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector.png" alt=""></a></li>
+                <div class="back">
+                    <img src="images/illustration-header-mobile.png" alt="">
+                </div>
```

Comment:
- [ ] Please use the image as a background defined on your CSS file as suggested by the previous reviewer. 
```
.introduction {
  other styles: here;
  background: url("images/illustration-header-mobile.png");
}
```
For more information [click here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image).